app/views/providers/income_summary/_add_other_income.html.erb

```diff
@@ -1,7 +1,7 @@
 <li>
   <h2 class="app-task-list__section">
     <span class="app-task-list__section-number">
-      <%= image_tag('plus_icon.svg') %>
+      <%= image_tag('plus_icon.svg', alt: 'Plus sign') %>
```

Comment:
ideally, `'Plus sign'` should be from locale but maybe it doesn't matter that much...